### ISSUE TITLE
[Drop zone] File upload button fails a11y contrast check

### ISSUE THREAD

**Issue by aveline** [Shopify/polaris-react-archive#4856]
## Issue summary

File upload button in the Drop zone component fails a11y contrast

![storybook polaris shopify com_](https://user-images.githubusercontent.com/3474483/147000850-36a769e8-895f-4308-b0f0-e35ca5f2ced1.png)

The accessibility automated tests are failing intermittently because of this

```
- FAIL: undefined
  Error: Elements must have sufficient color contrast
  Violating node(s):
    <div class="FileUpload-Action_30WpL">Add files</div>
```

## Expected behavior

There should be sufficient contrast for the button

## Steps to reproduce the problem

1. See component in Storybook https://storybook.polaris.shopify.com/?path=/story/all-components-drop-zone--drop-zone-with-file-upload
